ui-tests/src/test/java/org/openmrs/reference/page/HomePage.java

```diff
@@ -29,6 +29,7 @@
     private static final By FIND_PATIENT_RECORD = By.id("coreapps-activeVisitsHomepageLink-coreapps-activeVisitsHomepageLink-extension");
     private static final By DATA_MANAGEMENT = By.id("coreapps-datamanagement-homepageLink-coreapps-datamanagement-homepageLink-extension");
     private static final By APPOINTMENT_SCHEDULING = By.id("appointmentschedulingui-homeAppLink-appointmentschedulingui-homeAppLink-extension");
+    private static final By LOGGED_IN_USER = By.xpath("//*[@id='navbarSupportedContent']/ul/li[1]");
```

Comment:
if i am to accept an xpath, for this, i would start from the ul at-least or else use a better reference that may hardly break

Reply:
This has been addressed [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/454#discussion_r662772257)